site/content/docs/user/quick-start.md

```diff
@@ -59,25 +42,78 @@ On macOS via MacPorts:
 sudo port selfupdate && sudo port install kind
 {{< /codeFromInline >}}
 
-On macOS via Bash:
+On Windows via Chocolatey (https://chocolatey.org/packages/kind)
+{{< codeFromInline lang="powershell" >}}
+choco install kind
+{{< /codeFromInline >}}
+
+### Installing From Release Binaries
+
+Pre-built binaries are avilable on our [releases page](https://github.com/kubernetes-sigs/kind/releases).
+
+To install, download the binary for your platform from "Assets", then rename it to `kind` (or perhaps `kind.exe` on Windows) and place this
+into your `$PATH` at your preferred binary installation directory.
+
+On Linux:
+
+{{< codeFromInline lang="bash" >}}
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/{{< stableVersion >}}/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /some-dir-in-your-PATH/kind
+{{< /codeFromInline >}}
+
+On macOS:
 
 {{< codeFromInline lang="bash" >}}
 curl -Lo ./kind https://kind.sigs.k8s.io/dl/{{< stableVersion >}}/kind-darwin-amd64
 chmod +x ./kind
 mv ./kind /some-dir-in-your-PATH/kind
 {{< /codeFromInline >}}
 
-On Windows:
+On Windows in [PowerShell](https://en.wikipedia.org/wiki/PowerShell):
 
 {{< codeFromInline lang="powershell" >}}
 curl.exe -Lo kind-windows-amd64.exe https://kind.sigs.k8s.io/dl/{{< stableVersion >}}/kind-windows-amd64
 Move-Item .\kind-windows-amd64.exe c:\some-dir-in-your-PATH\kind.exe
 {{< /codeFromInline >}}
 
-On Windows via Chocolatey (https://chocolatey.org/packages/kind)
-{{< codeFromInline lang="powershell" >}}
-choco install kind
-{{< /codeFromInline >}}
+### Installing From Source
+
+In addition to the pre-built binary + package manager installation options listed
+above you can install from kind from source with `GO111MODULE="on" go get sigs.k8s.io/kind@{{< stableVersion >}}` or clone this repo
```

Comment:
`go install` is more preferable for recent versions of Go
https://github.com/golang/go/issues/43684

Reply:
There are notes about that just a few sentences below in the fully detailed section on installing from source with go, currently this command supports more versions however.